recipe/emailpassword/api/signin.go

```diff
@@ -29,6 +29,8 @@ func SignInAPI(apiImplementation epmodels.APIInterface, options epmodels.APIOpti
 		return nil
 	}
 
+	supertokens.InfoLoggerWithCode[int(supertokens.API_CALLED)]("SignInAPI")
```

Comment:
A few comments I added on the node PR apply here as well:

- I think the call log should be debug-level and the response info since the response log includes more information and it's more likely to be useful on its own.
- I'd prefer if both of these kinds of logs were done in a single place (supertokens middleware, etc)
- I'd prefer if the API implementation didn't care about the log level of these messages.